README.md

```diff
@@ -89,17 +89,255 @@ Name of the application pool.
+
+#### `identity_type`
+
+Specifies the account identity under which the application pool runs. Note: Starting in IIS 7.5 the default value is `:ApplicationPoolIdentity`. (In IIS 7.0 the default value was `:NetworkService`).  Valid options `:ApplicationPoolIdentity`, `:LocalService`, `:LocalSystem`, `:NetworkService`, or `:SpecificUser`
+
+#### `idle_timeout`
+
+Specifies how long (in minutes) a worker process should run idle if no new requests are received and the worker process is not processing requests. After the allocated time passes, the worker process should request that it be shut down by the WWW service
+
+#### `idle_timeout_action`
+
+Specifies the action to perform when the `idle_timeout` duration has been reached. Before IIS 8.5, a worker process that was idle for the duration of the `idle_timeout` property would be terminated. After IIS 8.5, you have the choice of terminating a worker process that reaches the `idle_timeout` limit, or suspending it by moving it from memory to disk. Suspending a process will likely take less time and consume less memory than terminating it. Valid options `:Terminate` or `:Suspend`
+
+#### `load_user_profile`
+
+Specifies whether IIS loads the user profile for the application pool identity. Setting this value to false causes IIS to revert to IIS 6.0 behavior. IIS 6.0 does not load the user profile for an application pool identity. Valid options `true` or `false`
+
+#### `log_event_on_process_model`
+
+Specifies which action taken in the process gets logged to the Event Viewer. In IIS 8.0, the only action that applies is the idle timeout action, in which the process is terminated because it was idle for the idleTimeout period
+
+#### `logon_type`
+Specifies the logon type for the process identity. (For additional information about logon types, see the LogonUser Function topic on Microsoft's MSDN Web site). Valid options `:LogonBatch` or `:LogonService`
+
+#### `manual_group_membership`
+
+Specifies whether the IIS_IUSRS group Security Identifier (SID) is added to the worker process token. When false, IIS automatically uses an application pool identity as though it were a member of the built-in IIS_IUSRS group, which has access to necessary file and system resources. When true, an application pool identity must be explicitly added to all resources that a worker process requires at runtime. Valid options `true` or `false`
+
+#### `max_processes`
+
+Indicates the maximum number of worker processes that would be used for the application pool.
+
+A value of '1' indicates a maximum of a single worker process for the application pool. This would be the setting on a server that does not have NUMA nodes.
```

Comment:
If the values in this list are integers, they should not have single quotes or backticks